[2023-12-10 19:39:35,860] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-16T00:00:00+00:00 [queued]>
[2023-12-10 19:39:36,002] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-16T00:00:00+00:00 [queued]>
[2023-12-10 19:39:36,012] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:39:36,038] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:39:36,045] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:39:36,618] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-03-16 00:00:00+00:00
[2023-12-10 19:39:36,632] {standard_task_runner.py:52} INFO - Started process 2499 to run task
[2023-12-10 19:39:36,720] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-03-16T00:00:00+00:00', '--job-id', '687', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpkg0nmtt0', '--error-file', '/tmp/tmpepvzm4je']
[2023-12-10 19:39:36,745] {standard_task_runner.py:80} INFO - Job 687: Subtask segunda_tarea
[2023-12-10 19:39:37,558] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-16T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:39:39,773] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-16T00:00:00+00:00
[2023-12-10 19:39:39,779] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:39:39,791] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:39:40,038] {subprocess.py:85} INFO - Output:
[2023-12-10 19:39:40,085] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:39:40,107] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:39:40,851] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230316T000000, start_date=20231210T193935, end_date=20231210T193940
[2023-12-10 19:39:41,070] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:39:41,684] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
